README.md

```diff
@@ -12,7 +12,7 @@ To run this application, you need to first clone the create-blazor-sparkline-cha
 
 ## Sparkline types
 
-Blazor Sparkline Charts support five types of sparklines ([line](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#line), [column](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#column), [area](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#area), [win-loss](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#winloss), and [pie](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#pie) to show data trends.
+Blazor Sparkline Charts support five types of sparklines ([line](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#line), [column](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#column), [area](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#area), [win-loss](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#winloss), and [pie](https://blazor.syncfusion.com/documentation/sparkline/chart-types?utm_source=github&utm_medium=listing&utm_campaign=blazor-sparkline-chart-github-samples#pie)) to show data trends.
```

Comment:
remove extra bracket
<img width="306" alt="image" src="https://github.com/SyncfusionExamples/create-blazor-sparkline-chart-in-blazor-server-app/assets/61148953/454776fd-12bb-44c1-b6b4-b0af2e8a3837">

Reply:
This bracket closes the opening bracket.